numba/cuda/dispatcher.py

```diff
@@ -191,6 +191,13 @@ def regs_per_thread(self):
         '''
         return self._codelibrary.get_cufunc().attrs.regs
 
+    @property
+    def const_mem_size(self):
+        '''
+        The size of user-allocated const memory for this kernel.
```

Comment:
What does "user-allocated" mean in this context?

Reply:
@gmarkall I am referring to uses of "cuda.const.array_like" in a kernel.